src/api/route-services/gem2s.js

```diff
@@ -166,7 +132,7 @@ class Gem2sService {
     // Make sure authJWT doesn't get back to the client
     delete messageForClient.authJWT;
 
-    await this.sendUpdateToSubscribed(experimentId, messageForClient, io);
+    await this.constructor.sendUpdateToSubscribed(experimentId, messageForClient, io);
```

Comment:
if this function is no longer static, do you still need to call `this.constructor`?

Reply:
This function is static, so we still need to call with `this.constructor`

Reply:
`sendUpdateToSubscribed ` is changed into a static method